app/controllers/admin/words_controller.rb

```diff
@@ -16,11 +16,46 @@ def create
 			render 'new'
 		end
 	end
+
+	def index
+		@category = Category.find(params[:category_id])
+		@word = @category.words
+
+		
+	end
+
+	def edit
+		@category = Category.find(params[:category_id])
+		@word = Word.find(params[:id])
+
+	end
+
+	def update
+		@word = Word.find(params[:id])
```

Comment:
@iori19960828 Add category variable here, to avoid errors when updating with validation error:
`@category = Category.find(params[:category_id])`